### DISCUSSION TITLE
DB migration from Kimai 1.x to Kimai 2.x

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#4181] [On-Premise: Installation & Updates]
Hi,
does anybody know how to migrate db data like users, projects, customers, timesheets, invoices and all these stuff to the new Kimai 2.x version?
I didn't want to make an upgrade of the old version but installed the new version itself  instead so I can test it and parallely the old Kimai can still be used for work.

Best regards
Klaus

**Comment by idemus** [comment]
Just found it...
https://www.kimai.org/documentation/migration-v1.html

**Reply by kevinpapst** [reply]
Be careful to not mix up Kimai 2 v1.x with the ancient Kimai 1:
https://www.kimai.org/documentation/versions.html
